timestamp-implementation-advice.xml

```diff
@@ -85,87 +85,129 @@
         <t>In this document we describe the properties of various available time values on modern operating systems, discuss the trade-offs of using one over the other, and provide guidance to help implementors make an informed choice with some real-life examples.</t>
      </section>
 
-      <section title="Keeping Time: Different Clocks">
-          <t> Because time is relative to an observer, there cannot be a
-              universally agreed upon time. At best we can achieve an
-              approximation by updating our own observed time with a
-              common reference time shared with other observers.</t>
-          <t> As this reference time is what we naively assume clocks on a
-              wall are showing, we shall call it the "wall time." For most
-              applications, it is based on the Universal Coordinated Time
-              (UTC), an international standard time determined by averaging
-              the output of several high-precision time-keeping devices.
-              However, as UTC is following Earth's solar time, it occasionally
-              needs to be adjusted through leap seconds.</t>
-          <t> An individual computer system's preception of time differs from
-              this idealized wall time. Staying close to it requires some
-              effort that comes with its own set of drawbacks. Systems
-              therefore provide access to different types of clocks with
-              different properties. Unfortunately, there is no standard
-              terminology and definitions for these types. For the purpose of
-              this document, we therefore define three different kinds of
-              clocks that a system may or may not provide.</t>
+    <section title="Keeping Time: Wall Time and Various Clocks">
+      <t>
+        Even if one ignores the intricacies introduced by the relative nature
+        for a Newtonian view of an absolute time ticking away at a constant
+        rate, the varying inaccuracies of all practical time-keeping devices
+        make agreeing on a precise time difficult.
+        An approximation can be achieved by first defining how to arrive
+        at a shared reference time and then updating one's own clocks every
+        now and then.
+      </t><t>
+        Such a reference time is called a time scale. There are several
+        scales, but the most commonly used one is the Universal
+        Coordinated Time (UTC). As it represents what we naively think every
+        clock on a wall should be showing, we shall refer to it as "wall
+        time."
+      </t><t>
+        One curious property of UTC is that because it attempts to keep in
+        sync with the Earth's varying rotation, occassional leap seconds are
+        required. As a consequence, the difference between two UTC time
+        values is not necessarily equal to the time passed.
```

Comment:
"attempts"?  Not "a measure addressing non constant rotation speed" or something?
Your writing style makes me smile, but is quite informal!